examples/remix-auth-email-link/app/models/User.ts

```diff
@@ -0,0 +1,9 @@
+export type User = {
+  // Add other fields (if any) here
+  email: string
+}
+
+export async function getUser({email}: {email: string}): Promise<User> {
+  // You should create/fetch this user from your db
+  return Promise.resolve({email})
+}
```

Comment:
Let's align this with other examples by putting this code in `app/data.server.ts`